.github/workflows/weekly-serverless.yml

```diff
@@ -11,12 +11,22 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       deployment_name: ${{ steps.set_deployment_name.outputs.date-name }}
+      stack_version: ${{ steps.set_deployment_name.outputs.stack-version }}
     steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Hermit Environment
+        uses: ./.github/actions/hermit
```

Comment:
i don't see anything new that requires it. what am i missing?

Reply:
In order to retrieve `ELK_VERSION`, we need to initialize Hermit.